xrdp/xrdp.ini.in

```diff
@@ -258,6 +258,26 @@ password=ask
 ; Currently NeutrinoRDP doesn't support dynamic resizing. Uncomment
 ; this line if you're using a client which does.
 #enable_dynamic_resizing=false
+;
+; perf.* setting can be used only with neutrinordp proxy.
+;
+; You can allow the experience setting in the RDP Client by uncommenting
```

Comment:
Suggest the following comment instead:-
```
; By default, performance settings requested by the RDP client are ignored
; and chosen by NeutrinoRDP. Uncomment this line to allow the user to
; select performance settings in the RDP client.
```

Reply:
We think it is easy to understand.
I accept your suggestion.